src/hashtable.c

```diff
@@ -1936,6 +1962,9 @@ int hashtableNext(hashtableIterator *iterator, void **elemptr) {
         }
         bucket *b = iter->bucket;
         if (iter->pos_in_bucket == 0) {
+            if (isPrefetchingValues(iter)) {
```

Comment:
nit: The wording here is a little confusing, implies it's already prefetching as opposed to the action it should take.
```suggestion
            if (shouldPrefetchValues(iter)) {
```

Alternatively, `isPrefetchValuesIterator` would also make more sense.

Reply:
Agree.